app/models/concerns/orchestration.rb

```diff
@@ -22,6 +22,14 @@ def rebuild_methods=(methods)
       @rebuild_methods = methods || {}
     end
 
+    def rebuild_methods_for(only = nil)
+      if only.present?
+        (@rebuild_methods || {}).select { |k,v| only.include?(k) || only.include?(v) }
+      else
+        @rebuild_methods || {}
+      end
+    end
```

Comment:
Why are you comparing both keys and values? To me only keys look better, they are more readable. Here is how accepted values look like at the moment:

```
[6] pry(main)> (Nic::Managed.rebuild_methods.values + Host::Managed.rebuild_methods.values + Nic::Managed.rebuild_methods.keys + Host::Managed.rebuild_methods.keys).join(', ')
=> "DHCP, DNS, TFTP, rebuild_dhcp, rebuild_dns, rebuild_tftp"
```

How about only accept "DHCP" "DNS" and "TFTP"?